src/otx/core/data/dataset/tile.py

```diff
@@ -124,12 +197,16 @@ def get_tiles(self, image: np.ndarray, item: DatasetItem) -> tuple[list[OTXDataE
         """
         tile_ds = DmDataset.from_iterable([item])
         tile_ds = tile_ds.transform(
-            Tile,
-            grid_size=self.tile_config.grid_size,
+            OTXTileTransform,
+            tile_size=self.tile_config.tile_size,
             overlap=(self.tile_config.overlap, self.tile_config.overlap),
             threshold_drop_ann=0.5,
         )
 
+        if self.dm_subset.name == "val":
```

Comment:
After considering https://github.com/openvinotoolkit/datumaro/pull/1264 and https://docs.ultralytics.com/yolov5/tutorials/tips_for_best_training_results/#dataset, I believe that evaluation on no label image is also important for demonstrating model's accuracy on false positives. Therefore, I think that we should not filter an image with empty annotation in validation/test subsets.

@jaegukhyun

Reply:
I'm not sure whether tiling has same strategy with normal detection training in OTX1.5.
Could you verify this?

Reply:
@jaegukhyun We've implemented a filtering mechanism for empty images in both **train** and **val** datasets (excluding the test set). You can find more details on the implementation at this link: [GitHub - OTX Detection Adapters Dataset Module](https://github.com/openvinotoolkit/training_extensions/blob/develop/src/otx/algorithms/detection/adapters/mmdet/datasets/dataset.py#L344). This idea was originally proposed by you in OTX 1.x. Initially, I included all tiles for evaluation, but during tile optimization exercises, we discovered that this approach was too slow. Implementing the filter significantly improved evaluation speed.

@vinnamkim If you believe this filter is unnecessary, I recommend bringing it up with Mark. This feature plays a crucial role in optimizing the speed of tiling. Removing it would likely result in slower tiling processes. Ultimately, the decision regarding whether slower training is acceptable rests with Mark.

Reply:
I agree on @eugene123tw, since OTX1.5 also filters validation dataset

Reply:
I'll discuss it with Wonju and Mark